.github/workflows/contributors-record.yml

```
@@ -0,0 +1,32 @@
+# Give your test a name!
+name: ContributorsRecord
+
+# Decide when to run the tests
+#
+# This configuration sets the test to run on pushes to master
+# and on pull requests that are opened to master
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

```suggestion
    branches:
      - master
    types:
      - opened
```
I think we might need this so that it's only run once per PR rather than on _every push_ to the PR

---

Good point.